common/app/views/fragments/atoms/snippet.scala.html

```diff
@@ -1,4 +1,4 @@
-@(className: String, label: String, headline: String, id: String, isAmp: Boolean)(body: Html)(implicit request: RequestHeader)
+@(className: String, label: String, headline: String, id: String, isAmp: Boolean)(body: Html)(footer: Html)(implicit request: RequestHeader)
```

Comment:
what's the benefit of having this footer parameter over just putting it in the body?

Reply:
good question. currently, absolutely none. will simplify that 😅